src/CustomComponents/NavBar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {FaBars} from 'react-icons/fa'
+import './NavBar.css';
+import Button1 from "./GenericButton/Button1";
+
+class NavBar extends React.Component{
+    render(){
+      
+        return(
+        <div>
+        <nav>
+            <ul>
+                <li class="logo">CryptoBara</li>
+                <li class="items"><a href="#">Home</a></li>
+                <li class="items"><a href="#">About</a></li>
+                <li class="items"><a href="#">Discord</a></li>
+                <li class="items"><a href="#">Feedback</a></li>
+                <li class="items"><Button1 description="Log In" height="40px" width="120px"></Button1></li>
```

Comment:
Missing href here. Don't know if thats intentional or not, but im assuming these buttons need to link us to somewhere

Reply:
It is intentional to keep the href out. There is nothing to link it to, thus ignored for now. The login and sign up popup page or a separate page should be implemented first.